cargo-dist/templates/installer/installer.ps1.j2

```diff
@@ -381,7 +381,8 @@ function Invoke-Installer($artifacts, $platforms) {
         Write-Information ""
         Write-Information "To add $dest_dir to your PATH, either restart your system or run:"
         Write-Information ""
-        Write-Information "    `$env:Path = `"$dest_dir;`$env:Path`""
+        Write-Information "    set Path=$dest_dir;%Path%   (cmd)"
```

Comment:
We don't have to escape `%Path%`, right? I guess that's not powershell syntax

Reply:
yeah powershell is very %percentsyntaxwho%